backend/src/meals.js

```diff
@@ -99,6 +99,110 @@ async function existsMeal(client, week) {
     return output
 }
 
+async function findClientMeal(client, week) {
+  let data
+  await Meal.findOne({clientID: client._id, startDate: week}, function(err, meal) {
+    if (err) {
+      console.log("error")
```

Comment:
Don't display clients with no meal data. Right now it's unable to handle past weeks. To test, try to print a week in April